lib/use-footer.js

```diff
@@ -1,22 +1,21 @@
 import { defaultPlacement } from '@neovici/cosmoz-dropdown';
 const _defaultPlacement = ['top-right', ...defaultPlacement];
 
-export const useFooter = ({
-	host,
-	...rest
-}) => {
+export const useFooter = ({ host, ...rest }) => {
 	const {
 			csvFilename = 'omnitable.csv',
 			xlsxFilename = 'omnitable.xlsx',
 			xlsxSheetname = 'Omnitable',
 			topPlacement = _defaultPlacement,
-		} = host;
+		} = host,
+		isMiniTable = host.hasAttribute('mini');
```

Comment:
This is not a great way to determine if the omnitable is in `mini` state. You should make use of [isMiniSize](https://github.com/Neovici/cosmoz-omnitable/blob/18c0da7d4dc8ffa8e11708f0fd529ab0271715b5/lib/use-mini.js#L5) and determine the number of toolbar items in the main hook.

```suggestion
      maxToolbarItems,
		} = host;
```